### ISSUE TITLE
Not able to cancel in display fingerprint

### ISSUE THREAD

**Issue by hariramnandagopal** [SelfLender/react-native-biometrics#56]
Hi, 

Thanks for your recent release, It works pretty well in most scenarios, however, I cant see the cancel button if the device has an in-display fingerprint (I am using OnePlus 6T). It seems to be after trying out version 2.0.0, i was able to cancel the in-display fingerprint in previous version [1.6.*]. 

could you please clarify if this was removed intentionally or more of a bug?

Thanks again.
Cheers.

**Comment by Aarshi-Kredx** [comment]
> Hi,
> 
> Thanks for your recent release, It works pretty well in most scenarios, however, I cant see the cancel button if the device has an in-display fingerprint (I am using OnePlus 6T). It seems to be after trying out version 2.1.1, i was able to cancel the in-display fingerprint in previous version [1.6.*].
> 
> could you please clarify if this was removed intentionally or more of a bug?
> 
> Thanks again.
> Cheers.

  I'm facing the same issue, did you find any resolution for this ?

**Comment by neoreact32** [comment]
No, I had to roll back to the previous version so that the user can cancel it,

**Comment by Aarshi-Kredx** [comment]
> No, I had to roll back to the previous version so that the user can cancel it,

Can you specify which version did you roll back to ?

**Comment by AnuragTiwari1** [comment]
facing it, with 2.1.4 release as well, any workaround without rollback

**Comment by Danushka50** [comment]
Facing same issue. Is there any solution for latest version.
